UI/src/view/linear_asset/laneModellingForm.js

```diff
@@ -0,0 +1,597 @@
+
+      eventbus.on('laneModellingForm: reload', function() {
+        reloadForm($('#feature-attributes'));
+      });
+
+      function setInitialForm() {
+        defaultFormStructure = formStructure;
+        currentFormStructure = mainLaneFormStructure;
+        isAddByRoadAddressActive = false;
+        lanesAssets.setCurrentLane(parseInt(_.min(_.map(self._assetTypeConfiguration.selectedLinearAsset.get(), function (lane) {
+          return _.head(_.find(lane.properties, function (property) {
+            return property.publicId == "lane_code";
+          }).values).value;
+        }))));
+      }
+
+      new FeedbackDataTool(feedbackModel, assetTypeConfiguration.layerName, assetTypeConfiguration.authorizationPolicy, assetTypeConfiguration.singleElementEventCategory);
+
+      eventbus.on(self.events('selected'), function () {
+        setInitialForm();
+        reloadForm(rootElement);
+      });
+
+      eventbus.on(self.events('unselect', 'cancelled'), function() {
+        rootElement.find('#feature-attributes-header').empty();
+        rootElement.find('#feature-attributes-form').empty();
+        rootElement.find('#feature-attributes-footer').empty();
+        dateutil.removeDatePickersFromDom();
+      });
+
+      eventbus.on('closeForm', function() {
+        rootElement.find('#feature-attributes-header').empty();
+        rootElement.find('#feature-attributes-form').empty();
+        rootElement.find('#feature-attributes-footer').empty();
+        dateutil.removeDatePickersFromDom();
+      });
+
+      eventbus.on('layer:selected', function(layer) {
+        if(self._assetTypeConfiguration.layerName === layer){
+          $('ul[class=information-content]').empty();
+
+          if(self._assetTypeConfiguration.isVerifiable)
+            self.renderLinkToWorkList(layer);
+          if(self._assetTypeConfiguration.hasInaccurate)
+            self.renderInaccurateWorkList(layer);
+        }
+      });
+
+      eventbus.on('application:readOnly', function(){
+        if(self._assetTypeConfiguration.layerName ===  applicationModel.getSelectedLayer() && self._assetTypeConfiguration.selectedLinearAsset.count() !== 0) {
+          setInitialForm();
+          reloadForm(rootElement);
+        }
+      });
+    };
+
+    self.renderAvailableFormElements = function(asset, isReadOnly, sideCode, setAsset, getValue, isDisabled, alreadyRendered) {
+      function infoLabel(publicId) {
+        var info;
+        var infoElement;
+
+        switch(publicId) {
+          case "lane_code":
+            info = 'Kaistan lisäys:';
+            break;
+          case "end_road_part_number":
+            info = 'Kaistan loppuu:';
+            break;
+          case "initial_road_number":
+            info = 'Kaistan alkaa:';
+            break;
+        }
+
+        if(info)
+          infoElement = $('<div class="form-group"><label class="control-label" style="text-align: left;">' + info + '</label></div>');
```

Comment:
Things like below it's to be put at CSS files, please.
```javascript
style="text-align: left;

Reply:
Done